clibabi32/clibabi32.rst

```diff
@@ -1581,6 +1585,11 @@ When \_AEABI\_PORTABILITY\_LEVEL != 0, the required definition of
 jmp\_buf cannot be used to create jmp\_buf objects. Instead, a jmp\_buf
 must be passed as a parameter or allocated dynamically.
 
+If Branch Target Identification mechanism is enabled, the
```

Comment:
As an alternative suggestion using "BTI setting" and "BTI clearing" from the v8-M Arm ARM version B.o

If the Branch Target Identification mechanism is enabled, the longjmp may transfer control to the destination stored in the jmp\_buf, using a BTI setting instruction that requires a BTI clearing instruction at the destination.

Reply:
I'm happy with such a wording, even though I personally interpret it pretty much as a requirement, rather than as a permission. Something like "Don't place BTIs on your own risk" :D